source/Applications/Basic/FileFormats/ReadIterateZDF.m

```diff
@@ -1,7 +1,12 @@
 % Import ZDF point cloud.
 
+% Adding directory that contains zividApplication to search path.
+addpath(genpath([fileparts(fileparts(fileparts(pwd))),filesep,'Camera',filesep,'Basic']));
+
+app = zividApplication;
+
 % The Zivid3D.zdf file has to be in the same folder as this sample script.
-Filename = 'Zivid3D.zdf';
+Filename = strcat(char(Zivid.NET.Environment.DataPath), '/Zivid3D.zdf');
```

Comment:
Filename = [char(Zivid.NET.Environment.DataPath), '/Zivid3D.zdf'];
is cleaner

Reply:
done